lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+  def initialize
+    @dishes = []
+  end
+
+  def dishes
+    @dishes = 
+    [
+      { name: 'Burger', price: 10.95  },
+      { name: 'Pizza', price: 14.00 },
+      { name: 'Salad', price: 7.60 },
+      { name: 'fries', price: 2.90 }
+    ]
+  end
+
+   def add_dish(name, price)
+   @dishes << {name: name, price: price}
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.